<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_recipe</class>
 <widget class="QMainWindow" name="choose_recipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>嘉然饿啦要吃饭！！！(/≧▽≦)/</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_jiaran.qrc">
    <normaloff>:/1/2.JPG</normaloff>:/1/2.JPG</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#choose_recipe{border-image: url(:/1/background.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="choose_recipe_back">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>0</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>时间</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="breakfast_choice">
         <property name="text">
          <string>早餐</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lunch_choice">
         <property name="text">
          <string>午餐</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QRadioButton" name="dinner_choice">
         <property name="text">
          <string>晚餐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="supper_choice">
         <property name="text">
          <string>夜宵</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>价格</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="minprice_choice">
         <property name="text">
          <string>下限要求</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minprice_box">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="maxprice_choice">
         <property name="text">
          <string>上限要求</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxprice_box">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>甜度（1~3）</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="sweet_choice">
         <property name="text">
          <string>有要求</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sweet_box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>辣度（1~3）</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="spicy_chioce">
         <property name="text">
          <string>有要求</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spicy_box">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>地点</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="place_choice">
         <property name="text">
          <string>无要求</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="qingzhen_choice">
         <property name="text">
          <string>清真食堂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="east_choice">
         <property name="text">
          <string>东食堂  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="south_choice">
         <property name="text">
          <string>南食堂  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="north_choice">
         <property name="text">
          <string>北食堂  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 255);
font: 12pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string>我选好啦！</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent; border - width:0; border - style:outset
font: 14pt &quot;华文新魏&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>嘉然今天吃什么呢？？</string>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>就决定是你啦！</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>菜名：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>时间：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地点：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>甜度：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>辣度：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>价格：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>类型：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>评分：</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="again">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 255);
font: 12pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string>再选一次！</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>食品种类</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QRadioButton" name="kind_choice">
         <property name="text">
          <string>无要求</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fastfood_choice">
         <property name="text">
          <string>快餐</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QRadioButton" name="noodle_choice">
         <property name="text">
          <string>面    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dumpling_choice">
         <property name="text">
          <string>带馅的（包子饺子馄饨)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rice_choice">
         <property name="text">
          <string>米饭  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="other_choice">
         <property name="text">
          <string>其他</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="confirmation">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>720</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 255);
font: 12pt &quot;华文彩云&quot;;
</string>
    </property>
    <property name="text">
     <string>就吃它了！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main_jiaran.qrc"/>
 </resources>
 <connections/>
</ui>
